.github/workflows/build-test.yml

```diff
@@ -556,15 +558,15 @@ jobs:
           check_name: Functional Test Report ${{matrix.os}}
         if: steps.test_functional_group.outcome == 'failure' && failure()
 
-      # Mosts tests are commented, hence nothing runs, thus no coverage (if not commented, CI falls over as this code expects coverage files)
-      # Upload unit test coverage reports for later use in the "reports" job.
-      - name: Upload unit test coverage reports
-        uses: actions/upload-artifact@v2
-        if: "(success() || failure()) && !contains(github.ref, 'refs/heads/release')"
-        with:
-          name: ${{runner.os}}-${{env.COVERAGE_REPORTS}}
-          path: .nyc_output
-          retention-days: 1
+    #   # Mosts tests are commented, hence nothing runs, thus no coverage (if not commented, CI falls over as this code expects coverage files)
+    #   # Upload unit test coverage reports for later use in the "reports" job.
```

Comment:
This throws error, and functional atests (old style) are almost non-existent, hence don't see the need to break CI just for coverage.
Once we've fixed our functional tests we  can re-vist this